server/server.go

```diff
@@ -54,6 +54,12 @@ type RaftLogListener interface {
 	Receive(*RaftLog)
 }
 
+// AuthzEnforcer contains a casbin enforcer and a lock, which is used to reload permissions safely
+type AuthzEnforcer struct {
+	enforcer  *casbin.Enforcer
+	authzLock *sync.RWMutex
```

Comment:
Is there a reason this is a mutex pointer? Generally I think it's preferred to just use a value _unless_ the mutex needs to be passed around (which I don't think is the case here?).

Reply:
Indeed, my bad.

Fixed